packages/components/src/Actions/ActionFile/ActionFile.scss

```diff
@@ -0,0 +1,14 @@
+input[type='file'] {
+	&.action-file-label {
```

Comment:
the `input[type='file']` qualifier is not necessary. `.action-file-label` is used as css module in your component